osu.Game/Database/DatabaseContextFactory.cs

```diff
@@ -14,6 +14,6 @@ public DatabaseContextFactory(GameHost host)
             this.host = host;
         }
 
-        public OsuDbContext GetContext() => new OsuDbContext(host.Storage.GetDatabaseConnectionString(@"client-ef"));
+        public OsuDbContext GetContext() => new OsuDbContext(host.Storage.GetDatabaseConnectionString(@"client"));

```

Comment:
So what happens when I run TestCasePlaySongSelect?

Reply:
Nothing has changed there; it uses a facade storage which resolves to an in-memory SQLite instance